packages/matchbox/src/components/EmptyState/Image.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledImage } from './styles';
+
+import { Picture } from '../Picture';
+
+const Image = React.forwardRef(function Image(props, userRef) {
+  const { src } = props;
+
+  return (
+    <StyledImage
+      display={['none', null, 'block']}
+      width="100%"
+      maxWidth="600px"
+      height="auto"
+      ref={userRef}
+    >
+      <Picture>
+        <Picture.Image src={src} />
```

Comment:
I would pass through `children` as well here, before `Picture.Image` in case we need to pass additional sources here

Reply:
Might also be a good idea to pass through `className`